src/util/StringUtils.h

```diff
@@ -8,6 +8,7 @@
 #include <absl/strings/str_replace.h>
 #include <unicode/bytestream.h>
 #include <unicode/casemap.h>
+#include <unicode/uloc.h>
```

Comment:
This include (which is probably not needed at all,) should be in the .cpp file. (Actually goes for all the other includes when the functions are only implemented in the .cpp file, you can clean those up while you are at it.

Reply:
Fixed, there were in general way too many includes.